default/AI/ColonisationAI.py

```diff
@@ -593,6 +602,12 @@ def get_colony_fleets():
 
     evaluated_outpost_planets = assign_colonisation_values(evaluated_outpost_planet_ids,
                                                            AIFleetMissionType.FLEET_MISSION_OUTPOST, None, empire)
+    # if outposted planet would be in supply range, let outpost value be best of outpost value or colonization value
+    for pid in set(evaluated_outpost_planets).intersection(evaluated_colony_planets):
+        if planet_supply_cache.get(pid, -99) >=0:
```

Comment:
Can `planet_supply_cache` contain negative values? Can it be replaced to `pid in  planet_supply_cache`?